app/configurations/config.hb.js

```diff
@@ -20,6 +20,12 @@ const maxLat = 48.64040;
 const minLon = 8.78597;
 const maxLon = 8.98613;
 
+// Matomo Tag Manager
```

Comment:
What is the advantage of having the matomo url split in it's constituents instead of just MATOMO_URL?

Reply:
if you deploy to staging, the hostname will be different.
in matomo you would track the staging server seperately, in e.g. configured in Matomo as "mib-staging". Then the path in the URL would change. The container_id would also be different.